indy_common/authorize/auth_map.py

```diff
@@ -175,13 +184,14 @@
            addNewSteward.get_action_id(): AuthConstraint(TRUSTEE, 1),
            addNewTrustAnchor.get_action_id(): AuthConstraintOr([AuthConstraint(TRUSTEE, 1),
                                                                 AuthConstraint(STEWARD, 1)]),
+           addNewNetworkMonitor.get_action_id(): AuthConstraint(STEWARD, 1),
```

Comment:
Not allowing trustees to create network monitors (while allowing stewards) seems weird